app/src/routes/login/login.vue

```diff
@@ -74,19 +75,23 @@ export default defineComponent({
 
 		onMounted(() => fetchProviders());
 
-		watch(providers, () => {
-			providerOptions.value = providers.value
-				.filter((provider) => !AUTH_SSO_DRIVERS.includes(provider.driver))
-				.map((provider) => ({ text: formatTitle(provider.name), value: provider.name }));
-			providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
-		});
-
 		return { t, te, authenticated, providers, providerSelect, providerOptions, provider, driver };
 
 		async function fetchProviders() {
 			try {
 				const response = await api.get('/auth');
 				providers.value = response.data.data;
```

Comment:
Sorry to add another change, but now we don't have the watch block you can completely kill this variable!

Reply:
I don't think that I can just remove the `providers` variable. It is returned in line 77 and used in sso-links.vue.
I did remove the `disableDefault` variable though, it is indeed useless now.

Reply:
Yeah, this was my bad. I didn't see that it got passed to the SSO component. You can resolve this. 😄